### ISSUE TITLE
Decide on Datastructure to save images for display on frontend

### ISSUE THREAD

**Issue by blueCat11** [Maxikilliane/DH_MusicAnalysis#31]
It's necessary to decide how we should save the images our backend generates during the analysis process. An example would be histograms for occurrence counts of chord roots.

Currently, I have a folder per session under media_root. In this folder (named after the session key) I generate a graphs folder, which can contain the graphs generated by that user. 
I can see several ways forward:

1. Decide on a basic naming scheme for files, pass the path to the frontend, display and provide possibility to download there. Generate new files everytime a user calls the necessary methods and pass the new paths. Possibly delete all the old files regularly.
2. Decide on a more elaborate naming scheme for files which takes into account for which piece of music they were generated. In case of a user request, check if the graph was already generated,  proceed to generate only if necessary, then pass the path, display/download... The files are unique to each user.
3. Use a more elaborate naming scheme for files as in 2., but don't save the generated graphs in the session folder (which ultimately gets deleted after a session expires). Instead, save them into a filestructure under the media_root, so that regeneration of graphs is minimized.

Basically it's a decision whether disk space is more important or time to run the generation.

A part of this decision, but a different issue:
Should we manage this using the database backend (that is to say, decide on models to represent those graphs etc) and use django ORM to handle saving to disk and eventually deleting, or should we do it by hand using os?

@Maxikilliane @RiaRentnueg  I could really use your input on this issue, I think we should meet to discuss this.
